orca-igor/src/main/groovy/com/netflix/spinnaker/orca/igor/tasks/GetCommitsTask.groovy

```diff
@@ -95,31 +94,35 @@ class GetCommitsTask implements DiffTask {
       String targetAmi = getTargetAmi(stage.context, region)
 
       //get commits from igor
-      sourceCommit = resolveCommitFromAmi(ancestorAmi, account, region)
-      targetCommit = resolveCommitFromAmi(targetAmi, account, region)
-
-      commitsList = []
+      sourceInfo = resolveInfoFromAmi(ancestorAmi, account, region)
```

Comment:
We already get the build information when we call `getByAmiId` below but we'd previously thrown everything away after extracting the commit hash from `appversion`.  I'm using the data returned by this call to also retrieve the build number.